lasagna-master/smunilla/lasagna_master.go

```diff
@@ -0,0 +1,42 @@
+package lasagna
+
+func PreparationTime(layers []string, avgTime int) int {
+	if avgTime == 0 {
+		avgTime = 2
+	}
+
+	return len(layers) * avgTime
+}
+
+func Quantities(layers []string) (int, float64) {
+	const noodleGrams = 50
+	const sauceLiters = 0.2
+
+	var noodlesLayers = 0
+	var sauceLayers = 0.0
+
+	for _, v := range layers {
+		if v == "noodles" {
+			noodlesLayers++
+		}
+
+		if v == "sauce" {
+			sauceLayers++
+		}
```

Comment:
An if/else block would be better but a switch block would be better-er IMO :wink: